### PR TITLE
Add Equipment, EquipmentType, and Equipment Manager

### PR DESCRIPTION
**Pull request by Bryan-BC**

### CODE REVIEW

File: src/main/java/seedu/Equipment/EquipmentManager.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class EquipmentManager {
+    private final HashMap<String, Equipment> equipmentList = new HashMap<>();
+
+    public void addEquipment(String itemName, String serialNumber, EquipmentType type, double cost, String purchasedFrom, String purchasedDate) {
+        Equipment equipment = new Equipment(itemName, serialNumber, type, cost, purchasedFrom, purchasedDate);
+        equipmentList.put(serialNumber, equipment);
```
Comment (shunyao643) [inline src/main/java/seedu/Equipment/EquipmentManager.java]: `putIfAbsent()` instead?

Comment (yuhaochua) [inline src/main/java/seedu/Equipment/EquipmentManager.java:11]: if the serialNumber already exists, should throw an error that will be caught when AddCommand.execute() is run

File: src/main/java/seedu/Equipment/EquipmentManager.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    public Equipment checkEquipment(String serialNumber) {
+        return equipmentList.get(serialNumber);
+    }
+
+    public ArrayList<Equipment> listEquipment(EquipmentType type) {
+        ArrayList<Equipment> listOfEquipments = new ArrayList<>();
+        for (Equipment equipment : equipmentList.values()) {
+            if (equipment.getType() == type) {
```
Comment (shunyao643) [inline src/main/java/seedu/Equipment/EquipmentManager.java]: Do we need `.equals` for this instead?

File: src/main/java/seedu/Equipment/EquipmentManager.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+public class EquipmentManager {
+    private final HashMap<String, Equipment> equipmentList = new HashMap<>();
+
+    public void addEquipment(String itemName, String serialNumber, EquipmentType type, double cost, String purchasedFrom, String purchasedDate) {
+        Equipment equipment = new Equipment(itemName, serialNumber, type, cost, purchasedFrom, purchasedDate);
+        equipmentList.put(serialNumber, equipment);
+    }
+
+    public Equipment checkEquipment(String serialNumber) {
```
Comment (yuhaochua) [inline src/main/java/seedu/Equipment/EquipmentManager.java]: Does it make more sense for user to search for equipment via name? If so then we would need to change this.

**Review by shunyao643 — CHANGES_REQUESTED** [review]
Hi, please take note of the change as discussed for `checkEquipment`: use `String itemName` instead.

File: src/main/java/seedu/Equipment/EquipmentManager.java (hunk 1 above)
Comment (yuhaochua) [inline src/main/java/seedu/Equipment/EquipmentManager.java]: if the serialNumber already exists, should we throw an error that will be caught when AddCommand.execute() is run?

**Pull request merged**